README.md

```diff
@@ -1,104 +1,2 @@
-    git clone https://github.com/Cloud-CV/EvalAI-ngx.git evalai-ngx && cd evalai-ngx
-    ```
-
-2. Build and run the Docker containers. This might take a while. You should be able to access EvalAI at `localhost:8888`.
-
-    ```
-    docker-compose up
-    ```
-
-3. That's it. Open web browser and hit the url [http://127.0.0.1:8888](http://127.0.0.1:8888). Three users will be created by default which are listed below -
-
-    **SUPERUSER-** username: `admin` password: `password`  
-    **HOST USER-** username: `host` password: `password`  
-    **PARTICIPANT USER-** username: `participant` password: `password`
-
-### For deploying with [Surge](https://surge.sh/):
-
-Surge will automatically generate deployment link whenever a pull request passes Travis CI. 
-
-Suppose pull request number is 123 and it passes Travis CI. The deployment link can be found here: `https://pr-123-evalai.surge.sh`
-
-## Code scaffolding
-
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
-
-## Code Documentation
-
-We are using [compodoc](https://compodoc.github.io/website/guides/jsdoc-tags.html) for documentation. The goal of this tool is to generate a documentation for all the common APIs of the application like modules, components, injectables, routes, directives, pipes and classical classes.
-
-Compodoc supports [these](https://compodoc.github.io/website/guides/jsdoc-tags.html) JSDoc tags.
-
-## Contributing
-
-Please go through our [Contribution Guidelines](https://github.com/Cloud-CV/EvalAI-ngx/blob/master/.github/CONTRIBUTING.md). Also go through our detailed [Code Structure Guide](https://github.com/Cloud-CV/EvalAI-ngx/blob/master/.github/CODE_STRUCTURE.md) to make the most of existing re-usable features. Finally, go through the [Pull Request Template](https://github.com/Cloud-CV/EvalAI-ngx/blob/master/.github/PULL_REQUEST_TEMPLATE.md) when creating your pull request.
-
-
-
-### Building and Serving the documentation
-
-Run the following command to build and serve the docs:
-```
-npm run doc:buildandserve
-```
-Open http://localhost:8080 in the browser to have a look at the generated docs.
-
-## Build
-
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
-
-## Running unit tests
-
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
-
-## Running end-to-end tests
-
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
-
-## The Team
-
-EvalAI-ngx is currently maintained by [Shekhar Rajak](http://s-hacker.info/), [Mayank Lunayach](https://github.com/lunayach), [Shivani Prakash Gupta](https://www.behance.net/shivaniprakash19), [Rishabh Jain](https://rishabhjain2018.github.io/) and [Deshraj Yadav](https://deshraj.github.io).
+Adds support for tsv and gz file in file submissions
```

Comment:
@rlee7 Why do we have to delete the contents of readme for this change? 

Can you please revert this